pilot/pkg/xds/ads.go

```diff
@@ -73,20 +73,20 @@ type Connection struct {
 
 	node *model.Proxy
 
-	// Sending on this channel results in a push. We may also make it a channel of objects so
-	// same info can be sent to all clients, without recomputing.
+	// Sending on this channel results in a push.
 	pushChannel chan *Event
 
-	LDSListeners []*listener.Listener                 `json:"-"`
-	RouteConfigs map[string]*route.RouteConfiguration `json:"-"`
-	CDSClusters  []*cluster.Cluster
-
-	// Both ADS and EDS streams implement this interface
+	// Both ADS and SDS streams implement this interface
 	stream DiscoveryStream
 
-	// Original node metadata, to avoid unmarshall/marshall. This is included
-	// in internal events.
+	// Original node metadata, to avoid unmarshall/marshall.
+	// This is included in internal events.
 	xdsNode *core.Node
+
+	// Computed Xds data. Mainly used for debug display.
```

Comment:
does anyone use this? I  would be fine with removing it, we already support /debug/config_dump anyways which is the same thing

Reply:
Not sure. But if no one is using would be good to remove